spring-cloud/spring-cloud-dapr/greeting/src/main/java/org/example/hello/GreetingController.java

```diff
@@ -0,0 +1,24 @@
+package org.example.hello;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/")
+public class GreetingController {
+    @RequestMapping(value = "/", method = RequestMethod.GET)
```

Comment:
Assuming it's supported in dapr, we should prefer the GetMapping annotation on these controller methods.